### PR TITLE
Kinetic panda move group python

### PR DESCRIPTION
**Pull request by mlautman**

### CODE REVIEW

File: doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py [hunk 1]
```diff
@@ -85,7 +86,7 @@ def move_group_python_interface_tutorial():
 
   ## Wait for RVIZ to initialize. This sleep is ONLY to allow Rviz to come up.
   print "============ Waiting for RVIZ..."
-  rospy.sleep(10)
+  rospy.sleep(1)
```
Comment (davetcoleman) [inline doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py]: I'm certainly not a fan of this... is the python and Rviz being launched in the same roslaunch file? if we separate them, i think we could remove this sleep

File: doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py [hunk 2]
```diff
@@ -110,27 +111,42 @@ def move_group_python_interface_tutorial():
...
+  joints = group.get_current_joint_values()
+  ## We will adjust some of the joint values
+  joints[1] = -pi/4
+  joints[3] = -pi/2
+  joints[5] = pi/3
+  ## The go command can be called with joint values, poses, or without any
+  ## parameters if you have already set the pose or joint target for the
+  ## group
+  group.go(joints, wait=True)
+  rospy.sleep(3)
```
Comment (davetcoleman) [inline doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py]: can we avoid this sleep with some kind of event driven wait?

File: doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py (hunk 1 above)
Comment (mlautman) [reply doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py]: They were being launched together but I already separated them out. I'm not a fan of this either. I was considering adding a line to wait for user input in between each step.

File: doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py (hunk 2 above)
Comment (mlautman) [reply doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py]: yerp

**Comment by mlautman** [comment]
Ready for second review. @davetcoleman I will deal with the merge conflicts when this is approved.

**Comment by mlautman** [comment]
This is being replaced by https://github.com/PickNikRobotics/moveit_tutorials/pull/15

**Pull request closed without merging**